hicommonwealth/commonwealth
# 7053 quill pasting bug

## Link to Issue
Closes: #7053 

## Description of Changes
- Changed the name of `useNotionPaste` to `useQuillPasteText` to be more accurate
- Using the Quill API finding the cursor position and pasting the content

## "How We Fixed It"

- Using the quill api, found the cursor position and pasting the content where the cursor is located 

## Test Plan
- Go to a thread or make a new one
- Ensure you can paste where you want

## Other Considerations

https://www.loom.com/share/306138c84d79400b8233dc7791f2a183
